heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/AppsV1beta1Controller.java

```diff
@@ -228,7 +228,7 @@ boolean isStatefulSet() {
     final Config runtimeConfiguration = getRuntimeConfiguration();
     final String[] executorCommand =
         SchedulerUtils.getExecutorCommand(configuration, runtimeConfiguration,
-            containerId, ports);
+            Integer.valueOf(containerId), ports);
```

Comment:
This needs to be a string because we use an environment variable for the container id

Reply:
will change back